site/content/en/docs/faq/_index.md

```diff
@@ -18,6 +18,19 @@ Example:
 minikube start --kubernetes-version=v1.15.0
 ```
 
+## How can I create more than one cluster with minikube?
+
+By default, minikube start creates a cluster named minikube. If you would like to create a different cluster or change its name, you can use the --profile (or -p) flag, which will create a cluster with the specified name.
```

Comment:
ad the backtick `minikube start`, `minikube` (the cluster name), `--profile` and `-p`

Reply:
done